clients/extension/src/sessions/mutations/useRemoveVaultSessionMutation.ts

```
@@ -0,0 +1,36 @@
+import { useMutation, UseMutationOptions } from '@tanstack/react-query'
+
+import { useInvalidateQueries } from '@lib/ui/query/hooks/useInvalidateQueries'
+import { appSessionsQueryKey, useVaultsAppSessionsMutation, useVaultsAppSessionsQuery } from '../state/appSessions'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)